pkg/network/bridge/lister.go

```diff
@@ -22,19 +25,36 @@ func (bl BridgeLister) GetResourceNamespace() string {
 }
 
 func (bl BridgeLister) Discover(pluginListCh chan dpm.PluginNameList) {
-	var plugins = make(dpm.PluginNameList, 0)
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		panic(err.Error())
+	}
+
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		panic(err.Error())
+	}
 
-	bridgesListRaw := os.Getenv(BridgesListEnvironmentVariable)
-	bridges := strings.Split(bridgesListRaw, ",")
+	for {
+		var plugins = make(dpm.PluginNameList, 0)
 
-	for _, bridgeName := range bridges {
-		if len(bridgeName) > maxBridgeNameLength {
-			glog.Fatalf("Bridge name (%s) cannot be longer than %d characters", bridgeName, maxBridgeNameLength)
+		configmap, err := clientset.CoreV1().ConfigMaps("kube-system").Get("device-plugin-network-bridge", metav1.GetOptions{})
```

Comment:
Instead of polling, we should use monitoring using list/watch. A tiny example: https://stackoverflow.com/questions/40975307/how-to-watch-events-on-a-kubernetes-service-using-its-go-client

Reply:
Thanks, didn't know that existed, will implement.